kolibri/utils/cli.py

```diff
@@ -14,10 +15,44 @@
 
 import kolibri  # noqa
 from kolibri import dist as kolibri_dist  # noqa
+from kolibri import dist_cext as kolibri_dist_cext
+
 sys.path = sys.path + [
     os.path.realpath(os.path.dirname(kolibri_dist.__file__))
 ]
 
+
+"""
+Get the directory of dist_cext.
+"""
+# Directory with the c modules
+dirname = os.path.dirname(kolibri_dist_cext.__file__)
+# Python version of current platform
+python_version = 'cp' + str(sys.version_info.major) + str(sys.version_info.minor)
+dirname = os.path.join(dirname, python_version)
+
+# Current platform
+platform = util.get_platform()
+# For Linux system, there could be abi tags 'm' and 'mu'
+if 'linux' in platform and int(python_version[2:]) < 33:
+    dirname = os.path.join(dirname, 'linux')
+    # encode with ucs2
+    if sys.maxunicode == 65535:
+        dirname = os.path.join(dirname, python_version+'m')
+    # encode with ucs4
+    else:
+        dirname = os.path.join(dirname, python_version+'mu')
+elif 'macosx' in platform:
+    platform = 'macosx'
```

Comment:
Some of the logic added to this file looks like it duplicates logic above -- can we combine/reuse?